app/controllers/newflow/social_auth_controller.rb

```diff
@@ -63,7 +66,7 @@ def oauth_callback
                 sign_up: view_context.link_to(I18n.t(:"login_signup_form.sign_up"), newflow_signup_path)
               )
             )
-          when :authentication_taken || :taken
+          when :authentication_taken || 'taken'
```

Comment:
Seems odd that one would be a string and the other a symbol?

Reply:
testing this, because it's still failing, but the error code is definitely 'taken', so hoping a string will cover the error code